tests/config_id/README.md

```diff
@@ -0,0 +1,12 @@
+## Open Enclave Init-time Configuration Interface.
+This document describes how developers can set CONFIGID and CONFIGSVN fields from the host side software at enclave initialization time and retrieve same information on enclave software side.
+[Overview of SGX CONFIGID and CONFIGSVN](https://github.com/openenclave/openenclave/blob/cd72fd7069488ba6f453c8f5f47bd9fd9a6e6c0d/docs/DesignDocs/InitTimeConfigurationInterface.md#sgx-configid-and-configsvn-overview).
```

Comment:
If this document is SGX-specific, then the filename, title of the document, and title of the PR should all have SGX in them to make it clear that this is narrowly scoped, not generally applicable to any TEE

Reply:
Also the PR title and description say it's a "readme" on how a developer can set config id information, but this PR is actually for a design doc not a readme.  (In my view, design docs are intended to be about OE internals.   Docs meant for developers using the OE should, I think, be under docs/GettingStartedDocs or under samples.)

Reply:
@manojrupireddy  - please update the title to say - design doc. Since this builds on the init-config interface, I think it is fine to put it here.

Reply:
@dthaler Merged suggested edits and update PR title & description.